### PR TITLE
docs(i18n): copy edits to internationalization cookbook

### PR DESCRIPTION
**Pull request by kapunahelewong**
Shannon's copy edits to Internationalization.

### CODE REVIEW

**Comment by Foxandxss** [comment]
@wardbell can we get this ASAP? I would like to work on the chapter for new features, soon.

File: public/docs/ts/latest/cookbook/i18n.jade [hunk 1]
```diff
@@ -11,11 +11,11 @@ a#top
     * [Add _i18n-..._ translation attributes](#translate-attributes)
     * [Handle singular and plural](#cardinality)
     * [Select among alternative texts](#select)
-    * [Use the **_ng-xi18n_ extraction tool** to create a translation source file](#ng-xi18n)
-    * [Translate](#translate)
+    * [Create a translation source file with the **_ng-xi18n_ extraction tool**](#ng-xi18n)
+    * [Translate los mensajes de texto](#translate)
```
Comment (wardbell) [inline public/docs/ts/latest/cookbook/i18n.jade]: This should stay in English: "Translate text messages".  The Spanish was cute but inconsistent. Let's fix the actual subhead too.

File: public/docs/ts/latest/cookbook/i18n.jade [hunk 2]
```diff
@@ -179,23 +183,24 @@ a#cardinality
...
     which specifies the
     <a href="http://unicode.org/reports/tr35/tr35-numbers.html#Language_Plural_Rules" target="_blank" title="Pluralization Rules">pluralization rules</a>.
 
 a#select
 :marked
-  ## Select
+  ## Select among alternative texts
   The application displays different text depending upon whether the hero is male or female.
-  These text alternatives require translation too.
+  These text alternatives require translation, too.
```
Comment (wardbell) [inline public/docs/ts/latest/cookbook/i18n.jade]: Why the comma? A comma in _that_ position indicates a pause. Would you pause as in "... _require translation ... ... ... too_"? Who speaks like that? Please remove it or explain.

File: public/docs/ts/latest/cookbook/i18n.jade [hunk 3]
```diff
@@ -272,15 +278,17 @@ code-example(language="sh" class="code-shell").
   npm run i18n -- --i18nFormat=xmb -p path/to/tsconfig.json
 :marked
   Note the `--` flag before the options. 
-  It tells _npm_ that every flag thereafter should be passed through to `ng-xi18n`.
+  It tells _npm_ to pass every flag thereafter to `ng-xi18n`.
```
Comment (wardbell) [inline public/docs/ts/latest/cookbook/i18n.jade]: Oh my. Why did I write the passive?  Thanks for fixing.

File: public/docs/ts/latest/cookbook/i18n.jade (hunk 2 above)
Comment (kapunahelewong) [inline public/docs/ts/latest/cookbook/i18n.jade]: My mistake. I thought that was the rule but on looking it up it looks like your way is the right way. Removed.

File: public/docs/ts/latest/cookbook/i18n.jade (hunk 1 above)
Comment (kapunahelewong) [inline public/docs/ts/latest/cookbook/i18n.jade]: Fixed.

**Pull request merged**